### PR TITLE
Add webUI acceptance tests for checking upload profile picture feature

### LINKED ISSUE #32850: webUI tests for personal general settings
Add webUI tests that check the content displayed on the personal general settings page:

- [ ] profile picture upload and display
- [x] set profile picture from existing cloud files (PR #33877)
- [x] full name display and change
- [x] display of groups that the user is a member of
- [x] display of federated cloud id
- [x] display  of ownCloud version

Note: there are already tests for display/setting email address, password and language.

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Add webUI acceptance test for checking upload profile picture(avatar)

## Related Issue

- Fixes #32850

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/lib/PersonalGeneralSettingsPage.php [hunk 1]
```diff
@@ -284,4 +280,66 @@ public function deleteProfilePicture(Session $session) {
...
+	}
+
+	/**
+	 * return if the "invalid image" message is visible
+	 *
+	 * @return void
+	 */
+	public function isFileUploadErrorMsgVisible() {
+		$this->waitTillElementIsNotNull($this->invalidImageErrorMsgXpath);
+		$errorMsg = $this->find('xpath', $this->invalidImageErrorMsgXpath);
```
Comment (individual-it) [inline tests/acceptance/features/lib/PersonalGeneralSettingsPage.php]: no need to do a `find` again `waitTillElementIsNotNull` returns `null` or the emelemt

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/webUIPersonalSettings/personalGeneralSettings.feature [hunk 2]
```diff
@@ -54,4 +54,22 @@ Feature: personal general settings
   Scenario: User deletes the existing profile picture
     Given the user has set profile picture to "testimage.jpg" from their cloud files
     When the user deletes the existing profile picture
-    Then the preview of the profile picture should not be shown in the webUI
\ No newline at end of file
+    Then the preview of the profile picture should not be shown in the webUI
+
+  Scenario: User uploads new prfile picture
```
Comment (phil-davis) [inline tests/acceptance/features/webUIPersonalSettings/personalGeneralSettings.feature]: s/prfile/profile

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@individual-it please review again

**Review by individual-it — APPROVED** [review]

**Comment by phil-davis** [comment]
@dpakach please backport

**Comment by dpakach** [comment]
> @dpakach please backport

Ok will do now

**Comment by dpakach** [comment]
Stable10 backport on #33916

**Pull request merged**